initializr-service/src/main/resources/application.yml

```diff
@@ -962,6 +962,12 @@ initializr:
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
           artifactId: spring-cloud-stream-binder-kafka
+        - name: Kafka Streams
+          id: cloud-stream-binder-kafka-streams
+          description: Stream processing microservices with Kafka Streams API
+          versionRange: 1.3.0.RELEASE
```

Comment:
That version range doesn't look right. What is that supposed to represent? `versionRange` is the Spring Boot availability.

Reply:
@snicoll I copied that from the regular Kafka binder right above. There it represents the current GA release of the kafka binder. I followed the same version in the kafka streams scenario also. Is that wrong?

Reply:
`versionRange` is the Spring Boot  version. If this is a new module, I doubt that there is support for it in Spring Boot `1.3.0.RELEASE`.

Reply:
Yes, this is a new module - Need Boot 1.5.7. I will update. Thanks!

Reply:
Does it work with Spring Boot  2?

Reply:
It does. For that, we need to use the corresponding `2.0.0.BUILD-SNAPSHOT` or the released milestones for this artifacts. Does that happen automatically if one chooses Boot 2 on the initializr?

Reply:
Thanks for the feedback. Yes, that happens automatically based on the version range, that's why I am asking if it works with Spring Boot 2. The version is managed by the spring  cloud BOM so whatever is defined there.